acmens.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
The `acmens.py` is not meant to be run directly. Can we remove this and revert the mode to 0644?

Reply:
I don't install this project in my system, but clone the repository, audit the source code, install the dependencies manually, and then run the `acmens.py`. I find it convenient to execute it directly without having to type `python3` before.

But if you don't want that, then revert the two commits (or rebase it) - it's your project.
```
git revert dce3e9227b92b8140ed3653251c1df2a83e9268e
git revert 042ee50382c2b6b0de6bacfacd9d29b92be9916f
```

Reply:
@Istador, the recommended way to use `acmens` is via `pip`.

If would like to continue using the repo version, I suggest something like this:

```bash
cd /path/to/acmens

# init python virtual environment
virtualenv --python=python3 .venv

# activate and get into the virtual environment
. .venv/bin/activate

# this will put the repo version of acmens in your PATH in the virtual environment
make develop

# now you can simply use with
acmens --account-key user.key --email mail@example.com --csr domain.csr > signed.crt
# also note that any changes you to acmens.py will be reflected instantly in the acmens program in your PATH.

# to get out of the virtual environment, do:
deactivate
```

- - -

I've cherry-picked the 'fallback to dns' change in the PR. Thank you for sending the patch.